packages/checkbox/test/typings/checkbox.types.ts

```diff
@@ -21,11 +24,14 @@ assertType<string>(checkbox.name);
 assertType<string>(checkbox.value);
 
 // Mixins
-assertType<ActiveMixin>(checkbox);
-assertType<ElementMixin>(checkbox);
-assertType<CheckedMixin>(checkbox);
-assertType<DelegateFocusMixin>(checkbox);
-assertType<SlotLabelMixin>(checkbox);
+assertType<ActiveHost>(checkbox);
+assertType<ElementHost>(checkbox);
+assertType<FocusHost>(checkbox);
+assertType<DisabledHost>(checkbox);
```

Comment:
These are no longer available through `ActiveHost`. The main change in this PR is to use classes instead of interfaces to have protected methods, and the classes can't have multiple inheritance.